src/content/platform-integration/ios/setup.md

```diff
@@ -0,0 +1,254 @@
+    ```
+
+1. Start the Simulator with the following command:
+
+    ```console
+    $ open -a Simulator
+    ```
+
+{% endtab %}
+{% tab "Physical device" %}
+
+Set up each iOS device on which you want to test.
+
+### Configure your physical iOS device {:#attach .no_toc}
+
+1. Attach your iOS device to the USB port on your Mac.
+
+1. On first connecting an iOS device to your Mac,
+   your device will display the **Trust this computer?** dialog.
+
+1. Click **Trust**.
+
+   ![Trust Mac][]
+
+### Enable Developer Mode {: .no_toc}
+
+Apple requires enabling **[Developer Mode][]**
+on the device to protect against malicious software.
+
+1. Tap on **Settings** <span aria-label="and then">></span>
+   **Privacy & Security** <span aria-label="and then">></span>
+   **Developer Mode**.
+
+1. Tap to toggle **Developer Mode** to **On**.
+
+1. Restart the device.
+
+1. When the **Turn on Developer Mode?** dialog appears,
+   tap **Turn On**.
+
+### Create a developer code signing certificate {: .no_toc}
+
+To send your app to a physical iOS device,
+_even_ for testing, you must establish trust
+between your Mac and the device.
+In addition to trusting the device when that
+popup appears, you must upload a signed
+developer certificate to your device.
+
+To create a signed development certificate,
+you need an Apple ID.
+If you don't have one, create one.
+You must also enroll in the [Apple Developer][] program
+and create an [Apple Developer account][]. If you
+are just _testing_ your app on an iOS device,
+a personal Apple Developer account is free and works.
+
+:::note Apple Developer program
+When you want to _deploy_ your app to the
+App Store, you will need to upgrade your
+personal Apple Developer account to
+a professional account.
+:::
+
+### Prepare the device {: .no_toc}
+
+Find your **Profiles & Device Management** menu under
+**Settings**. (The exact location can vary by device.)
+Toggle your certificate to **Enable**.
+
+:::note
+If you can't find **VPN & Device Management** menu,
+run your app on your iOS device once, then try again.
+:::
+
+1. Under the **Developer App** heading,
+   you should find your certificate.
+
+1. Tap the certificate.
+
+1. Tap **Trust "&lt;certificate&gt;"**.
+
+1. When the dialog displays, tap **Trust**.
+
+If the **codesign wants to access key...** dialog appears:
+
+1. Enter your macOS password.
+
+1. Tap **Always Allow**.
+
+[Trust Mac]: /assets/images/docs/setup/trust-computer.png
+[Developer Mode]: {{site.apple-dev}}/documentation/xcode/enabling-developer-mode-on-a-device
+[Apple Developer]: {{site.apple-dev}}/programs/
+[Apple Developer account]: {{site.apple-dev}}/account
+
+{% endtab %}
+{% endtabs %}
+
+---
+
+
+Follow the instructions in the output for software
+you might need to install or further tasks to perform.
+
+## Start developing for iOS {: #start-developing}
+
+**Congratulations.**
+You can now start developing and testing Flutter apps on iOS!
+
+<div class="card-grid link-cards">
```

Comment:
Just as a note, I added these cards to correspond to the ones I added in the equivalent Android guide. Feel free to remove them or make changes.